packages/by-name/cli-release/package.nix

```diff
@@ -11,9 +11,10 @@
 (contrast.overrideAttrs (
   _finalAttrs: previousAttrs: {
     prePatch = ''
-      install -D ${lib.getExe genpolicy} cli/cmd/assets/genpolicy
-      install -D ${contrast.settings}/genpolicy-settings.json cli/cmd/assets/genpolicy-settings.json
-      install -D ${contrast.rules}/genpolicy-rules.rego cli/cmd/assets/genpolicy-rules.rego
+      install -D ${lib.getExe genpolicy} cli/genpolicy/assets/genpolicy
+      install -D ${contrast.settings}/genpolicy-settings.json cli/genpolicy/assets/genpolicy-settings.json
+      install -D ${contrast.rules}/genpolicy-rules.rego cli/genpolicy/assets/genpolicy-rules.rego
+      install -D ${genpolicy.src}/src/kata-opa/allow-all.rego cli/genpolicy/assets/allow-all.rego
```

Comment:
Do we really need to embed the allow-all policy into the release cli? Are we planning to do a release with this policy?

Reply:
We should not release in this state, yes. If I just remove the file here, the release build will fail - is that acceptable? Alternatively, we could fail `NewConfig` for unsupported platforms (by checking for an empty embedded ruleset, for example).

Reply:
Ah, I thought the build wouldn't fail due to the placeholder, but we don't include that in src.